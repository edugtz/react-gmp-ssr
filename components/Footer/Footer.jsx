import React from "react";

import styles from "./Footer.module.scss";

const Footer = () => {
    return (
        <footer className={styles.footer}>
            <span className={styles['app-title']}>
                <b>netflix</b>roulette
      </span>
        </footer>
    );
};

export default Footer;
