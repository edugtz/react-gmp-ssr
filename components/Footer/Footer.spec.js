import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'

describe('Footer component', () => {
    it('should render correctly', () => {
        const component = renderer.create(<Footer />).toJSON()
        expect(component).toMatchSnapshot()
    })
})
