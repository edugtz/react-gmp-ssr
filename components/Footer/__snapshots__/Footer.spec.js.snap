// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<footer
  className="footer"
>
  <span
    className="app-title"
  >
    <b>
      netflix
    </b>
    roulette
  </span>
</footer>
`;
